### PR TITLE
Create sorensht.md

### PR DESCRIPTION
**Pull request by sorensht**
rawgit--https://rawgit.com/sorensht/sorensht.github.io/add-sorensht-profile/#!pages/vi/profiles/sorensht.md
I am Swornim Shrestha i am currently reading bsc.csit in Samriddhi college addressed in Bhaktapur, Nepal. I love doing programming. I love riding bikes and playing football.

### CODE REVIEW

**Comment by akshayuncc** [comment]
@sorensht  Hi, Looks Good .
Could you add another label and tell us about yourself a bit more. For example your interests, your skill, you could also add an image, etc. Please use as many Markdowns as possible no restrictions. The more markdowns you use the faster the PR approval rate.
All the best.

**Comment by singharpita** [comment]
Please use some markdown as well.

File: pages/vi/profiles/sorensht.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# User name: sorensht.md
+# rawgit link: https://rawgit.com/sorensht/sorensht.github.io/add-sorensht-profile/#!pages/vi/profiles/sorensht.md
```
Comment (dogi) [inline pages/vi/profiles/sorensht.md]: @sorensht by the way I would remove this two lines above

**Comment by akshayuncc** [comment]
@sorensht Hey good job you are getting the hang of markdowns. 
I am sharing the image of the file you updated. Could you just adjust it and add an image to it after removing the two lines mentioned by @dogi .
No worries you are doing a good job.

soren

**Comment by sorensht** [comment]
screen shot of my updated profile please have a look.

**Pull request merged**